### ISSUE TITLE
add verifyToken to volos-swagger 

### ISSUE THREAD

**Issue by borisirota** [apigee-127/volos#52]
Hi,

In order to implement an oauth2 server only (it's only purpose is to generate and verify tokens) based on volos-ouath the missing part is to expose the verifyToken operation

I guess that this method will expect the token and the required scopes to be inside the request body.

The use case for that is microservices architecture where one server is responsible for the token generation and verification and all the others are business logic only.

To support this it requires:
1. Add one more path (verify) to x-a127-services:oauth2:options:tokenPaths in swagger.yaml.
2. Add the specification of the method [here](https://github.com/apigee-127/volos/blob/master/swagger/spec/oauth_operations.yaml).
3. add a controller [here](https://github.com/apigee-127/volos/blob/master/swagger/lib/controllers/volos-oauth.js).
4. change the implementation of [the authenticate method](https://github.com/apigee-127/volos/blob/master/oauth/common/lib/oauth-connect.js#L86) or add a new one that expects the parameters to be in request's body.

Thanks,
Boris

**Comment by theganyo** [comment]
I agree that something like this seems to be a good enhancement. I'll look into it.
